pkg/cmd/server/start/bootstrap_node.go

```diff
@@ -212,12 +212,20 @@ func (o NodeOptions) loadBootstrap(hostnames []string, nodeConfigDir string) err
 
 	// try to refresh the latest node-config.yaml
 	o.ConfigFile = filepath.Join(nodeConfigDir, "node-config.yaml")
-	config, err := c.Core().ConfigMaps("kube-system").Get("node-config", metav1.GetOptions{})
+	config, err := c.Core().ConfigMaps(o.NodeArgs.BootstrapConfigNamespace).Get(o.NodeArgs.BootstrapConfigName, metav1.GetOptions{})
```

Comment:
doesn't seem like the namespace should be configurable.  Name yes, but I don't see why we'd let people change the namespace.